string request_type
string ip_address
int64 param_number
int64 value
string command_string
---
string response
